### PR TITLE
Add bazel_skylib 1.1.1

### PR DESCRIPTION
**Pull request by jsharpe**

### CODE REVIEW

File: modules/bazel_skylib/1.1.1/MODULE.bazel [hunk 1]
```diff
@@ -0,0 +1,11 @@
+module(
+    name = "bazel_skylib",
+    version = "1.1.1",
+    compatibility_level = 1,
```
Comment (jsharpe) [inline modules/bazel_skylib/1.1.1/MODULE.bazel:4]: @meteorcloudy is this the correct value for compatibility_level? I've not yet fuly understood the semantics of when this should be bumped.

**Comment by jsharpe** [comment]
This isn't actually the version of skylib that I want but its the closest to it at the moment before I can have a go at porting rules_foreign_cc to bzlmod. We're waiting on https://github.com/bazelbuild/bazel-skylib/issues/336 for an updated skylib release as we currently depend on a commit on main in the latest rules_foreign_cc.

**Comment by alexeagle** [comment]
Related #41
Who is responsible to update these, the rule maintainer or the registry maintainers...

File: modules/bazel_skylib/1.1.1/MODULE.bazel (hunk 1 above)
Comment (Wyverald) [reply modules/bazel_skylib/1.1.1/MODULE.bazel:4]: this should be bumped sparingly (e.g. when bazel_skylib 2.x is out).

**Comment by Wyverald** [comment]
> Who is responsible to update these, the rule maintainer or the registry maintainers...

The answer should be the maintainers of this module in the registry. This group could intersect with the rule maintainer and/or the registry maintainer, but the maintainers nominated in the module's metadata.json should be the ones to update the module. (Or not -- if nobody needs a certain version, maybe it's fine that it's not in the registry.)

**Pull request merged**